### DISCUSSION TITLE
what is the purpose of the new "device type" option ?

### DISCUSSION THREAD

**Discussion by avichou** [sannidhyaroy/Soduto#16] [Q&A] [answered]
what difference does it make whether I select desktop or laptop for example ?

**Comment by sannidhyaroy** [comment, marked as answer]
For now, there’s probably no difference as the icons are also the same in KDE Connect but that's KDE Connect's issue that they implemented the same icon for both desktops and laptops. By default, Soduto comes with Desktop mode, so KDE Connect will recognize the device as a desktop. It's upto the KDE Connect community, how they want to handle the differences between laptops and desktops. I added the dropdown, just because there was a device type variable that KDE Connect accepts.

**Comment by sannidhyaroy** [comment]
I think the latest versions of KDE Connect shows appropriate icons of device types.
